### ISSUE TITLE
Could not write the contents of a file downloaded using python requests into .nc file

### ISSUE THREAD

**Issue by Omkar20895** [lewismc/podaacpy#5]
```
try:    
     url = 'http://podaac.jpl.nasa.gov/ws/extract/granule/?datasetId=PODAAC-ASOP2-25X01&shortName=ASCATA-L2-25km&granuleName=ascat_20130719_230600_metopa_35024_eps_o_250_2200_ovw.l2.nc&bbox=45,0,180,90&format=netcdf'
    granule = requests.get(url)
    if granule.status_code == 404 or granule.status_code == 400 or granule.status_code == 503 or granule.status_code == 408: 
        granule.raise_for_status()

    file = open(granuleName, 'w+')
    file.write(granule.text.encode("utf-8"))

except requests.exceptions.HTTPError as e:
    print e
```

When I try to execute the above code it creates a local file with the name ascat_20130719_230600_metopa_35024_eps_o_250_2200_ovw.l2_subsetted_.nc

But when I try to open it using the panoply tool(which is used to view .nc files) I am getting an error 'There was an error opening the dataset java.lang.NegativeArraySizeException'.

**Comment by lewismc** [comment]
You need to checkout how to subset data and/or transfer it using possibly the NetCDF Python libraries.

**Comment by Omkar20895** [comment]
Lewis I am really sorry to say this but I couldn't find anything on this issue. I posted the issue in stackoverflow also but did not get any solution.

**Comment by lewismc** [comment]
I don't believe there is not a solution 😜
I'll find a solution and post here shortly.

On Friday, May 20, 2016, Omkar <notifications@github.com> wrote:

> Lewis I am really sorry to say this but I couldn't find anything on this
> issue. I posted the issue in stackoverflow also but did not get any
> solution.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/lewismc/podaacpy/issues/5#issuecomment-220583446>
>


-- 
*Lewis*

**Comment by lewismc** [comment]
In the meantime move on to the MCC file. This is all trivial Python code
and should not be presenting any issue at all.

On Friday, May 20, 2016, Omkar <notifications@github.com> wrote:

> Lewis I am really sorry to say this but I couldn't find anything on this
> issue. I posted the issue in stackoverflow also but did not get any
> solution.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/lewismc/podaacpy/issues/5#issuecomment-220583446>
>


-- 
*Lewis*

**Comment by Omkar20895** [comment]
Okay :D

**Comment by lewismc** [comment]
Did you check how this was being done in other data source modules?

**Comment by Omkar20895** [comment]
I am checking it out with some of my friends at Indian Meteorological Institute. They too work with the netcdf files. He said he would get back to me since he had very less idea about that. Thanks. :)

**Comment by Omkar20895** [comment]
@lewismc have you used ncgen tool anytime to convert ascii into netcdf data? Thanks

**Comment by Omkar20895** [comment]
@lewismc I have been working on this issue in a single direction and didn't explore other options. Aha, you just need to write two lines with the module **urllib** and _et voila_! 😄 

I am opening a PR for solving this. Thanks.

**Comment by lewismc** [comment]
Thanks

On Friday, June 10, 2016, Omkar <notifications@github.com> wrote:

> Closed #5 <https://github.com/lewismc/podaacpy/issues/5>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/lewismc/podaacpy/issues/5#event-688320416>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ABHJl9U9VD8wubxYnhoq4OtFRWTKxC5fks5qKSregaJpZM4Ign8u>
> .
>


-- 
*Lewis*
